pallet-chainlink-feed/src/lib.rs

```diff
@@ -63,6 +63,8 @@ pub mod pallet {
 		pub latest_round: RoundId,
 		pub first_valid_round: Option<RoundId>,
 		pub oracle_count: u32,
+		pub debt: Balance,
+		pub max_debt: Balance,
```

Comment:
Please document new fields.
I'd also suggest we fix missing docs for the other fields here, since a lot of them are used in the `create_feed` extrinsic

Reply:
I'd like to fix the docs in this PR, but my English sentence seems not smoothly enough LOL